tools/scripts-config-cdn/modules.json

```diff
@@ -185,15 +213,15 @@
   "@talend/react-stepper": {
     "var": "TalendReactStepper",
     "versions": {
-      ">= 5.27.0 || >= 6.0.0-alpha.10": {
+      ">= 5.27.0": {
         "development": "/dist/TalendReactStepper.js",
         "production": "/dist/TalendReactStepper.min.js"
       }
     },
     "style-versions": {
       ">= 5.27.0 < 5.31.0": {
         "development": "/dist/TalendReactStepper.css",
-        "production": "/dist/TalendReactStepper.css"
+        "production": "/dist/TalendReactStepper.min.css"
```

Comment:
No before/after here.
What for version >= 5.31.0 ?

Reply:
```suggestion
        "production": "/dist/TalendReactStepper.css"
```

Reply:
indeed, fixed thanks !